keras/backend/mxnet_backend.py

```diff
@@ -5345,15 +5276,24 @@ def __init__(self, *args, **kwargs):
 
             self.name = kwargs['name']
 
+            # if context in kwargs, remove it before calling super
```

Comment:
Add a link to bug to have context on why this is required to be removed from kwargs and then add back. What happens if it is not done?

Reply:
was trying to avoid changing front end keras code, so less conflict during rebase. But looks like official keras already added kwargs option. I made the change on front end now and will remove it during next merge with upstream.

reference: https://github.com/keras-team/keras/blob/master/keras/engine/network.py#L149